composer.json

```diff
@@ -15,20 +15,20 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "^2.7 || ^3.0 || ^4.0"
+        "symfony/framework-bundle": "^4.0 || ^5.0"
```

Comment:
the new event exists since 4.3, so this should be it I suppose.

Reply:
You know those moments when something is so obvious after the fact, that you bury your face in your hands? :wink: